src/odemis/gui/main.xrc

```diff
@@ -2970,667 +2971,635 @@
     <bg>#333333</bg>
     <style>wxWANTS_CHARS</style>
   </object>
-  <object class="wxPanel"
```

Comment:
No actually, not because of XRCed. I reformatted the file. There are gaps in code, and there are several parts of code which are improperly formatted. Now I have put back the older version (i hope i did so correctly), but i would still suggest to nicely format it, maybe from the master branch directly.